docs/manual-instrumentation.md

```diff
@@ -0,0 +1,115 @@
+# Manual instrumentation
+
+For most users, the out-of-the-box instrumentation is completely sufficient and nothing more has to
+be done.  Sometimes, however, users wish to add attributes to the otherwise automatic spans,
+or they might want to manually create spans for their own custom code.
+
+## Contents
+
+- [Manual instrumentation](#manual-instrumentation)
+- [Dependencies](#dependencies)
+  * [Maven](#maven)
+  * [Gradle](#gradle)
+- [Adding attributes to the current span](#adding-attributes-to-the-current-span)
+- [Creating spans around methods with `@WithSpan`](#creating-spans-around-methods-with---withspan-)
+  * [Suppressing `@WithSpan` instrumentation](#suppressing---withspan--instrumentation)
+  * [Creating spans around methods with `otel.instrumentation.methods.include`](#creating-spans-around-methods-with-otelinstrumentationmethodsinclude)
+- [Creating spans manually with a Tracer](#creating-spans-manually-with-a-tracer)
+
+# Dependencies
+
+> :warning: starting with 0.6.0, and prior to version 1.0.0, `opentelemetry-javaagent-all.jar`
```

Comment:
I think it's long enough ago now, safe to remove the "starting with"
```suggestion
> :warning: prior to version 1.0.0, `opentelemetry-javaagent-all.jar`
```

Reply:
If it's phrased "prior to version 1.0.0" and we hit 1.1 doesn't that mean that section isn't applicable any longer?   Maybe we should say "starting with 1.0.0", or did I completely misread it?

Reply:
Oh, nm, I misread it.  I'll take this suggestion.